couchdbkit/client.py

```diff
@@ -513,6 +513,8 @@ def get_rev(self, docid):
         @return rev: str, the last revision of document.
         """
         response = self._request_session.head(self._database_path(docid))
+        if response.status_code == 404:
+            raise ResourceNotFound
```

Comment:
It's been a while since I looked at the `requests` library, but I think that the usual way is to do this is:

```
try:
    response.raise_for_status()
except 404:
   #  handle/raise own exception
```

Reply:
updated